ui/app/src/models/GlobalState.ts

```diff
@@ -115,6 +114,9 @@ export interface GlobalState {
     currentUrl: string;
     computedUrl: string;
     showToolsPanel: boolean;
+    toolsPanelPageStack: {
+      pages: any[];
+    };
```

Comment:
Are we thinking this may have additional props? Why not it being directly the array of "pages"

Reply:
yes, I think is better to apply it directly to toolsPanelPageStack